### PR TITLE
Fix/boot resolution

### PR DESCRIPTION
**Pull request by gordominossi**

### CODE REVIEW

File: configuration/menu.lua [hunk 1]
```diff
@@ -225,15 +224,15 @@ local function reboot()
...
+    gui.bigButton(location.x, location.y+graphics.context().height-5, "Run", switchRunStatus)
+    gui.bigButton(location.x+5, location.y+graphics.context().height-5, "Save", save)
+    gui.bigButton(location.x+11, location.y+graphics.context().height-5, "Reboot", reboot)
+    gui.bigButton(location.x+19, location.y+graphics.context().height-5, "Shell", interrupt)
+    gui.bigButton(location.x+26, location.y+graphics.context().height-5, "Settings", configScreen, {location.x+2*selectionBoxWidth+2, location.y, graphics.context().width-(location.x+2*selectionBoxWidth+2), graphics.context().height-5, "NIDAS Settings", menu})
+    if updateAvailable() then gui.smallButton(graphics.context().width-21, graphics.context().height, "Update available!", update) end
+    gui.smallLogo(graphics.context().width-20, graphics.context().height-4, require("nidas_version"))
     component.gpu.fill(1, 1, 160, 50, " ")
     renderer.update()
 end
```
Comment (gordominossi) [inline configuration/menu.lua:238]: Calls take a while, could put them all in a variable

File: configuration/menu.lua [hunk 2]
```diff
@@ -8,9 +8,6 @@ local screen = require("lib.utils.screen")
 
 local component = require("component")
 
-graphics.setContext()
-local maxWidth = 160
-local maxheight = 50
```
Comment (gordominossi) [inline configuration/menu.lua:13]: Right here

File: configuration/menu.lua (hunk 1 above)
Comment (S4mpsa) [reply configuration/menu.lua:238]: True
I just find&replaced the old variable with the call.

Comment (gordominossi) [reply configuration/menu.lua:238]: Bad practice

**Pull request merged**